EcommerceApp/backend/spring-boot-ecommerce-app/src/main/java/kkramarenko/ecommerceapp/service/CheckoutServiceImpl.java

```diff
@@ -62,17 +75,35 @@ public PurchaseResponse placeOrder(Purchase purchase) {
         Customer existingCustomer = customerRepository.findCustomerByFirstNameAndLastNameAndEmail(
                     customer.getFirstName(), customer.getLastName(), customer.getEmail());
 
+        boolean discountRateChanged = false;
+        String newDiscountRate = "";
+
         if(existingCustomer != null){
```

Comment:
Code formatting:
`if (existingCustomer != null) {`

Reply:
You also can try to autoformat your code by using this feature in IDE: https://www.jetbrains.com/help/idea/reformat-and-rearrange-code.html#reformat_file

Reply:
Done